example-problems/array-sum/generators/main.cpp

```diff
@@ -4,10 +4,10 @@
 int main() {
     testgen::TestgenSession sess = testgen::init();
     int num_cnt = sess.gen.next_range(1, 10001);
```

Comment:
Looks like hardcoded and uncommented something

Reply:
Do you propose moving constants from generator code to some config file?

Reply:
I propose at least commenting this constant and not copy-pasting it three times

Reply:
If you do, please open new issue, because it definitely is topic for discussion.